testserver/minecraft.js

```diff
@@ -0,0 +1,68 @@
+// Require Node.js standard library function to spawn a child process
+var spawn = require('child_process').spawn;
```

Comment:
You should use `const` for unchanging variables like this.

Reply:
Done!